samples/interactive-tutorials/user_environment_setup.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Copyright 2022 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# set the Google Cloud project Id
+project_id=$1
+echo Project ID: $project_id
+gcloud config set project project_id
+
+timestamp=$(date +%s)
+
+service_account_id="service-acc-"$timestamp
+echo Service Account: $service_account_id
+
+# create service account (your project_id+timestamp)
```

Comment:
I think this `(your project_id+timestamp)` is incorrect compare to line 24: `"service-acc-"$timestamp`

Reply:
fixed